### PR TITLE
Topics proposal updates [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by samdutton**
To match the following proposed changes:
• [github.com/patcg-individual-drafts/topics/pull/80/files](https://github.com/patcg-individual-drafts/topics/pull/80/files)
• [github.com/patcg-individual-drafts/topics/pull/81/files](https://github.com/patcg-individual-drafts/topics/pull/81/files)

**DO NOT MERGE**

### CODE REVIEW

File: site/en/docs/privacy-sandbox/topics/index.md [hunk 1]
```diff
@@ -312,6 +303,62 @@ const creative = await response.json();
...
+Including the `{observe: false}` argument means that topics can be returned, but the call will not
+cause the current page to be included in the weekly epoch calculation, nor will it update the list
+of topics observed for the caller.
+
+### Use a header to access topics
+
+Instead of calling `document.browsingTopics()`, topics can be retrieved via a request header, and
+marked as observed and eligible for topics calculation via response headers.
+
+This is likely to be considerably more performant than using the JavaScript API.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/topics/index.md:325]:
```suggestion
This is likely much more performant than using the JavaScript API.
```

File: site/en/docs/privacy-sandbox/topics/index.md [hunk 2]
```diff
@@ -312,6 +303,62 @@ const creative = await response.json();
...
+
+The request header will be sent on document requests if the list of topics is non-empty and the
+request is allowable: for example, the appropriate [permission policy](#site-opt-out) is in play,
+and the context is secure.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/topics/index.md:329]:
```suggestion
The request header will be sent on document requests, if the list of topics is non-empty and the
request is allowable. For example, when the appropriate [permission policy](#site-opt-out) is in play,
and the context is secure.
```

File: site/en/docs/privacy-sandbox/topics/index.md [hunk 3]
```diff
@@ -312,6 +303,62 @@ const creative = await response.json();
...
+
+#### Request header example
+
+``` text
+Sec-Browsing-Topics: 123;model=1;taxonomy=1;version=2, 2;model=1;taxonomy=1;version=2
+```
+This header includes two topics from the
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/topics/index.md:336]:
```suggestion

This header includes two topics from the
```

File: site/en/docs/privacy-sandbox/topics/index.md [hunk 4]
```diff
@@ -312,6 +303,62 @@ const creative = await response.json();
...
+[taxonomy](https://github.com/patcg-individual-drafts/topics/blob/main/taxonomy_v1.md), 123 and 2,
+along with their version information.
+
+A Topics request header can also be provided with a `fetch()` request:
+
+``` javascript
+fetch(<url>, {browsingTopics: true})
+```
+#### Response header example
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/topics/index.md:345]:
```suggestion

#### Response header example
```

File: site/en/docs/privacy-sandbox/topics/index.md [hunk 5]
```diff
@@ -312,6 +303,62 @@ const creative = await response.json();
...
+#### Notes
+
+-  Redirects will be followed, and the topics sent in the redirect request will be specific to
+    the redirect URL.
+-  The request header will not modify state for the caller unless there is a corresponding
+    response header. That is, the topic of the page won't be considered observed, nor will it
+    affect the user's topic calculation for the next epoch.
+-  The response header will only be honored if the corresponding request included the topics
+    header (or would have included the header if it wasn't empty).
+-  The registrable domain used for topic observation is that of the URL of the request.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/topics/index.md:360]:
```suggestion
-  The URL of this request is the registrable domain used for topic observation.
```

File: site/en/docs/privacy-sandbox/topics/index.md [hunk 6]
```diff
@@ -312,6 +303,62 @@ const creative = await response.json();
...
+
+#### Notes
+
+-  Redirects will be followed, and the topics sent in the redirect request will be specific to
+    the redirect URL.
+-  The request header will not modify state for the caller unless there is a corresponding
+    response header. That is, the topic of the page won't be considered observed, nor will it
+    affect the user's topic calculation for the next epoch.
+-  The response header will only be honored if the corresponding request included the topics
+    header (or would have included the header if it wasn't empty).
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/topics/index.md:359]:
```suggestion
-  The response header is only honored if the corresponding request included the topics
    header (or would have included the header, if the request wasn't empty).
```

**Comment by samdutton** [comment]
Thank you @heyawhite! Excellent suggestions.

**Review by heyawhite — APPROVED** [review]

**Comment by samdutton** [comment]
Just a reminder to anyone seeing this: **do not merge**.

**Comment by heyawhite** [comment]
@samdutton would you prefer I revoke my approval?

Regardless, no one outside of me from the content team should touch this this week.

**Comment by samdutton** [comment]
> @samdutton would you prefer I revoke my approval?

No — that's fine. I just need to make sure I don't accidentally merge this myself ;).

**Pull request merged**